apps/api/src/app/environments/render/email-schema-extender.ts

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable no-param-reassign */
+import { TipTapNode } from '@novu/shared';
+
+// Define the command interface
+// eslint-disable-next-line @typescript-eslint/naming-convention
+interface ExpendEmailEditorSchemaCommand {
+  schema: TipTapNode;
+}
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please extend from `BaseCommand`.‏